src/FirstSteps/RPG/Heroes/Hero.cs

```diff
@@ -37,81 +38,7 @@ public void DisplayStats()
             .AddItem("Agility", _agility, Color.Yellow)
             .AddItem("Health", _health, Color.Red)
             .AddItem("Damage", _damage, Color.Green));
-            Console.WriteLine($"Equipment:\ntotal price: {DisplayTotalPrice()}\ntotal items: {DisplayTotalItems()}\ntotal weight; {DisplayTotalWeight()} ");
-        }
-
-        public bool AddItemToBackpack(Item item)
```

Comment:
The encapsulation approach is applicable here also. `AddItemToBackpack` should stay as the hero's method and you have to rewrite just a small part related to the backpack itself and replace it with `_equipment.TryAddItemToBackpack`.